condarecipe/larray-editor/larray-editor.json

```diff
@@ -0,0 +1,22 @@
+{
+    "menu_name": "LArray",
+    "menu_items":
+        [
+            {
+                "name": "LArray Editor",
+                "pywscript": "${PYTHON_SCRIPTS}/larray-editor-script.pyw",
+                "icon": "${MENU_DIR}/larray.ico",
+                "quicklaunch": true
+            },
+            {
+                "name": "LArray Documentation",
+                "webbrowser": "http://larray.readthedocs.io/en/stable",
+                "icon": "${MENU_DIR}/larray-help.ico"
```

Comment:
where does this icon come from? Do we have the rights to use it? Is it in the public domain?